Mendelugram/Mendelugram/View/Comment/CommentTableViewCell.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  CommentTableViewCell.swift
+//  Mendelugram
+//
+//  Created by Pacek on 21/04/2019.
+//
+
+import UIKit
+
+class CommentTableViewCell: UITableViewCell {
+
+    static let reuseIdentifier = String(describing: CommentTableViewCell.self)
```

Comment:
If you want to have it like this for sake of simplicity then no problem but it would be nicer if you defined this in an UITableViewCell extension or a protocol with protocol extenstion